vlogsApp/vlogsApp/Controller/GamingScreenViewController.swift

```diff
@@ -77,20 +111,31 @@ class GamingScreenViewController: UIViewController {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
 }
 
 //MARK: -
 
 extension GamingScreenViewController: UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+    
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        
         guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: BlogCollectionViewCell.identifier, for: indexPath) as? BlogCollectionViewCell else {
             return UICollectionViewCell()
         }
+        
         let blog = dataSource[indexPath.item]
-            cell.titleLabel.text = blog.title
-            cell.descriptionLabel.text = blog.description
+        
+        firebaseManager?.dowloadPhoto(path: blog.image, completion: { imageData in
+            cell.postImageView.image = UIImage(data: imageData)
+        })
+
+        cell.titleLabel.text = blog.title
```

Comment:
use a config function for configuration. You should call that function inside the completioner of downloadPhoto function.